packages/engine/tests/initializeEngine.test.ts

```diff
@@ -0,0 +1,56 @@
+import { initializeEngine, shutdownEngine } from '../src/initializeEngine'
+import { Engine } from '../src/ecs/classes/Engine'
+import assert from 'assert'
+import { EngineSystemPresets, InitializeOptions } from '../src/initializationOptions'
+
+/**
+ * tests
+ */
+describe('InitializeEngine', () => {
```

Comment:
i dont think this is really needed. it is touching too many global scope, we can revisit in the future